### PR TITLE
[rdma] Test PFC watchdog under many to one traffic pattern

### PR DESCRIPTION
**Pull request by baiwei0427**
Signed-off-by: Wei Bai webai@microsoft.com

### Description of PR

Summary: Test PFC watchdog under many to one traffic pattern
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?
We need more PFC watchdog test cases

#### How did you do it?
1. Add a new traffic pattern "many to one" in tests/ixia/pfcwd/files/pfcwd_multi_node_helper.py
2. Add a new function __gen_m2o_traffic in tests/ixia/pfcwd/files/pfcwd_multi_node_helper.py to generate traffic, including a PFC pause storm sent from port_id, and many data traffic items sent to port_id.
3. Add a new test function test_pfcwd_m2o in tests/ixia/pfcwd/test_pfcwd_m2o.py

#### How did you verify/test it?
I did test using SONiC switches and IXIA chassis. The Tgen API version is 0.0.75. The IXIA Linux API server version is 9.10.

#### Any platform specific information?
N/A

#### Supported testbed topology if it's a new test case?
T0 topology using IXIA chassis as the fanout

### CODE REVIEW

**Comment by baiwei0427** [comment]
retest this please

File: tests/ixia/pfcwd/test_pfcwd_m2o.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    ixia_api_serv_user, ixia_api_serv_passwd, ixia_api, ixia_testbed
+from tests.common.ixia.qos_fixtures import prio_dscp_map, all_prio_list,\
+    lossless_prio_list, lossy_prio_list
+
+from files.pfcwd_multi_node_helper import run_pfcwd_multi_node_test
+
+@pytest.mark.topology("tgen")
+
+@pytest.mark.parametrize("trigger_pfcwd", [True, False])
+def test_pfcwd_m2o(ixia_api,
```
Comment (neethajohn) [inline tests/ixia/pfcwd/test_pfcwd_m2o.py:16]: please rename to 'test_pfcwd_many_to_one'

**Review by neethajohn — APPROVED** [review]

**Review by neethajohn — APPROVED** [review]

**Pull request merged**